h2o-algos/src/main/java/hex/gam/MatrixFrameUtils/GamUtils.java

```diff
@@ -93,6 +92,12 @@ public static GLMParameters copyGAMParams2GLMParams(GAMParameters parms, Frame t
     setParamField(parms, glmParam, true, field2);
     glmParam._train = trainData._key;
     glmParam._valid = valid==null?null:valid._key;
+    glmParam._nfolds = 0; // always set nfolds to 0 to disable cv in GLM.  It is done in GAM
+    glmParam._fold_assignment = Model.Parameters.FoldAssignmentScheme.AUTO;
+    glmParam._keep_cross_validation_fold_assignment = false;
+    glmParam._keep_cross_validation_models = false;
+    glmParam._keep_cross_validation_predictions = false;
+    glmParam._is_cv_model = false; // disable cv in GLM.
```

Comment:
False by default again, even if not explicitly stated. No need to do it explicitly.

Reply:
@pavel: In this case, I am copying GAM parameters to GLM parameters. If I want cross-validation and set nfolds > 1, _is_cv_model will be true and therefore after the copying of parameters, GLM parameter _is_cv_model will also be true. Hence, I need to manually set them.